msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java

```diff
@@ -202,6 +223,23 @@ public static AcquireTokenOperationParameters createAcquireTokenOperationParamet
         return acquireTokenOperationParameters;
     }
 
+    public static ClaimsRequest addClientCapabilitiesToClaimsRequest(ClaimsRequest cr, String clientCapabilities){
+
+        ClaimsRequest mergedClaimsRequest = null;
```

Comment:
nit : instantiation and assignment can be combined

`final ClaimsRequest mergedClaimsRequest = (cr == null) ? new ClaimsRequest() : cr;`

Reply:
Done.

Reply:
Done